docs/formats.rst

```diff
@@ -250,6 +250,18 @@ The ``import_set()`` method also supports a ``skip_lines`` parameter that you
 can set to a number of lines that should be skipped before starting to read
 data.
 
+The ``export_set()`` method supports a ``column_width`` parameter. Depending 
+on the value passed, the column width will be set accordingly. It can be 
+either ``None``, an integer, or default "adaptive". If "adaptive" is passed, 
+the column width will be unique and will be calculated based on values' length. 
+For example:: 
```

Comment:
Let's remove those trailing spaces:
```suggestion
The ``export_set()`` method supports a ``column_width`` parameter. Depending
on the value passed, the column width will be set accordingly. It can be
either ``None``, an integer, or default "adaptive". If "adaptive" is passed,
the column width will be unique and will be calculated based on values' length.
For example::
```